examples/server/server.cpp

```diff
@@ -3281,14 +3280,17 @@ struct server_context {
                 }
 
                 // construct the speculation batch
-                slot.batch_spec.clear();
-                slot.batch_spec.add_text(id, slot.n_past, slot.id, true);
+                //slot.batch_spec.clear();
+                //slot.batch_spec.add_text(id, slot.n_past, slot.id, true);
```

Comment:
I think these comments can be safely removed since we no longer have notion of `common_batch` in the code base, these calls are specific to `common_batch`